Makefile

```diff
@@ -0,0 +1,10 @@
+all: grpc
+
+grpc:
```

Comment:
consider adding the proto as a prerequisite prerequisites

Reply:
https://www.gnu.org/software/make/manual/make.html#Rule-Introduction

Reply:
done